connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorPluginsResourceTest.java

```diff
@@ -182,29 +174,24 @@
     @Mock
     private Plugins plugins;
     private ConnectorPluginsResource connectorPluginsResource;
+    @Captor
```

Comment:
We don't use these annotations in the other tests. Do you think we should not use them here to keep tests consistent? WDYT?